build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+cd $(dirname $0)
+src=$(pwd)
+cd - >/dev/null
```

Comment:
Adding in quotation to prevent splitting and `exit` in case the `cd` fails.
```suggestion
cd "$(dirname "$0")" || exit 1
src="$(pwd)"
cd - >/dev/null || exit 1
```